### DISCUSSION TITLE
Adding a Chat option

### DISCUSSION THREAD

**Discussion by ava-silver** [sandboxnu/office-hours#691] [Ideas]
It would be nice to have an option to chat with the student once a TA starts to help them, just in case they aren't responding on teams.

**Comment by willstenzel** [comment]
Hi @ava-silver, thanks for creating this discussion. When a student doesn't respond do you think it's more often because they are afk or that they don't have the teams app open?

**Reply by ava-silver** [reply]
In the experience I had that caused me to open this ticket, the student didn't have teams open, as in teams said they were offline. They very well could have been afk, but I had no way of knowing because I couldn't contact them.
